src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -248,9 +248,16 @@ protected int doWork() {
         final FingerprintChecker checker = new FingerprintChecker(HAPLOTYPE_MAP);
         checker.setReferenceFasta(REFERENCE_SEQUENCE);
 
-        SequenceUtil.assertSequenceDictionariesEqual(SAMSequenceDictionaryExtractor.extractDictionary(inputPath), SAMSequenceDictionaryExtractor.extractDictionary(genotypesPath), true);
-        SequenceUtil.assertSequenceDictionariesEqual(SAMSequenceDictionaryExtractor.extractDictionary(inputPath), checker.getHeader().getSequenceDictionary(), true);
-
+        try {
```

Comment:
I didn't change these because they use the extra `checkPrefixOnly` argument, which the others don't. So rather than create another overload in Picard, I left these as direct calls to htsjdk. If you think its worth it though I could add a second method for those cases though.

Reply:
ahh, I see.  No I think that's fine it that case.